cheroot/connections.py

```diff
@@ -184,9 +186,10 @@ def stop(self):
     def run(self, expiration_interval):
         """Run the connections selector indefinitely.
 
-        Args:
-            expiration_interval (float): Interval, in seconds, at which
-                connections will be checked for expiration.
+        :param expiration_interval: Interval, in seconds, at which \
```

Comment:
I'd prefer that unrelated refactorings be submitted separately to keep the PR diff of a digestible size.

Reply:
Okay makes sense. I can open a new PR to apply this syntax consistently across the script. Just for my understanding, when not using `r"""`, the backslashes need to be doubled, right? So this was missing as well.

Reply:
Yep, there was some corner case with escaping EOL in docstrings related to this. So this is why they need to be turned into raw-strings.